plugins/CorePluginsAdmin/Commands/ListPlugins.php

```diff
@@ -37,20 +37,30 @@ protected function doExecute(): int
             });
         }
 
-        $plugins = array_map(function ($plugin) use ($pluginManager) {
-            return array(
+        $verbose = $this->getOutput()->isVerbose();
+
+        $plugins = array_map(function ($plugin) use ($pluginManager, $verbose) {
+            $pluginInformation = array(
                 '<info>' . $plugin . '</info>',
                 $pluginManager->isPluginBundledWithCore($plugin) ? 'Core' : 'Optional',
                 $pluginManager->isPluginActivated($plugin) ? 'Activated' : '<comment>Not activated</comment>',
```

Comment:
If a plugin can't be found in filesystem anymore we could actually write something else than `Not activated`:

```suggestion
                !$pluginManager->isPluginInFilesystem($plugin) ? '<error>Not found</error>' : ($pluginManager->isPluginActivated($plugin) ? 'Activated' : '<comment>Not activated</comment>'),
```